pallets/runtime/develop/src/runtime.rs

```diff
@@ -227,6 +229,20 @@ pub type DealWithFees = SplitTwoWays<
     Author<Runtime>, // 1 part (20%) goes to the block author.
 >;
 
+pub struct WeightToFee;
+impl WeightToFeePolynomial for WeightToFee {
+    type Balance = Balance;
+
+    fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+        smallvec![WeightToFeeCoefficient {
+            degree: 1,
+            coeff_frac: Perbill::from_percent(10),
+            coeff_integer: 0u128, // Coefficient is zero
+            negative: false,
+        }]
```

Comment:
Does this imply 10_000_000 weight -> 1 POLYX?

Reply:
Yes. It is